dd-sdk-android/src/test/kotlin/com/datadog/android/rum/assertj/ErrorEventAssert.kt

```diff
@@ -95,6 +97,82 @@ internal class ErrorEventAssert(actual: ErrorEvent) :
         return this
     }
 
+    fun hasUserInfo(expected: UserInfo?): ErrorEventAssert {
+        assertThat(actual.usr?.id)
+            .overridingErrorMessage(
+                "Expected RUM event to have usr.id ${expected?.id} " +
+                    "but was ${actual.usr?.id}"
+            )
+            .isEqualTo(expected?.id)
+        assertThat(actual.usr?.name)
+            .overridingErrorMessage(
+                "Expected RUM event to have usr.name ${expected?.name} " +
+                    "but was ${actual.usr?.name}"
+            )
+            .isEqualTo(expected?.name)
+        assertThat(actual.usr?.email)
+            .overridingErrorMessage(
+                "Expected RUM event to have usr.email ${expected?.email} " +
+                    "but was ${actual.usr?.email}"
+            )
+            .isEqualTo(expected?.email)
+        return this
+    }
+
+    fun hasConnectivityInfo(expected: NetworkInfo?): ErrorEventAssert {
+        val expectedStatus = if (expected?.isConnected() == true) {
+            ErrorEvent.Status.CONNECTED
+        } else {
+            ErrorEvent.Status.NOT_CONNECTED
+        }
+        val expectedInterfaces = when (expected?.connectivity) {
```

Comment:
Why not using the extension methods here to convert : `toErrorConnectivity`

Reply:
Because this is a test method ;)

Reply:
Ok ...I see you are testing those methods here...hmm...I guess in this case cause they are only used/coupled with these events it can work. Usually I am not a big fan of this approach cause if we remove say this part of the code and those methods would be used somewhere else we will be missing the unit tests completely for those.